files/commonfiles/class/Common/FileChecker.php

```
@@ -144,18 +144,16 @@ public static function setFilePermissions($target, $mode = 0777)
-        break;
+if ($op == 'copyfile') {
+    if (\Xmf\Request::hasVar('original_file_path', 'POST')) {
+        $original_file_path = $_POST['original_file_path'];
+    }
+    if (\Xmf\Request::hasVar('file_path', 'POST')) {
+        $file_path = $_POST['file_path'];
+    }
+    if (\Xmf\Request::hasVar('redirect', 'POST')) {
+        $redirect = $_POST['redirect'];
+    }
+    $msg = FileChecker::copyFile($original_file_path, $file_path) ? \constant('CO_' . $moduleDirNameUpper . '_' . 'FC_FILECOPIED') : \constant('CO_' . $moduleDirNameUpper . '_' . 'FC_FILENOTCOPIED');
```

PHPStan (2.1.46)

[error] Variable $file_path might not be defined.


(variable.undefined)